Dockerfile

```diff
@@ -4,6 +4,7 @@
 FROM golang:1.8 AS builder
 ENV DOCKER_HOST tcp://172.17.0.1:2375
 ENV LOCAL_REGISTRY 172.17.0.1
+ENV SKIP_SLOW_TESTS true
```

Comment:
you are pussy @ivanilves !

Reply:
that's bulling @vonrabbe :trollface: 

I'm so offended!